tests/tests.js

```diff
@@ -1,14 +1,42 @@
 const tape = require('tape');
-const request = require('supertest');
+const supertest = require('supertest');
 const server = require('../server.js');
+const request = require('request');
 
 tape('home route test: GET request to /', t => {
-  request(server)
+  supertest(server)
     .get('/')
     .end((err, res) => {
-      if (err) throw err;
+      t.error(err, 'No Erro');
```

Comment:
you don't need to provide a message here:
see https://www.npmjs.com/package/tape#terrorerr-msg

Reply:
is it preferred to use like an if statement and t.fail after it in case of an error  ?

Reply:
just `t.error(err)` is easier